docs/src/man/other-formats.md

```diff
@@ -71,6 +71,34 @@ makedocs(
 )
 ```
 
+## [Custom LaTeX style](@id custom-latex)
+
+### Load custom packages
+
+We have loaded many common packages in LaTeX,
+such as `fontspec`, `amsmath`, `listings`, `minted`, `tabulary`, `graphicx`,
+and more detailed configurations can be found in [`documenter.sty`](https://github.com/JuliaDocs/Documenter.jl/blob/master/assets/latex/documenter.sty).
+
+Users can load more custom packages by adding a `custom.sty` to the `assert/` folder,
+and the custom style (`custom.sty`) will be loaded after the default style (`documenter.sty`).
+
+### Custom preamble
+
+If you wish to fully customize the package loading, you need to write a custom preamble.
+
+The default preamble is currently defined in [`preamble.tex`](https://github.com/JuliaDocs/Documenter.jl/blob/master/assets/latex/).
```

Comment:
```suggestion
The default preamble is currently defined in [`preamble.tex`](https://github.com/JuliaDocs/Documenter.jl/blob/master/assets/latex/preamble.tex).
```

It's fine that linkcheck fails here in the PR as it will be fixed by the merge.